### PR TITLE
Fix WebServiceListener and WebServiceSender related warnings in config HTTP

### PR DESCRIPTION
**Pull request by Bibi0803**
1 - WebServiceListener calling webservices via de ServiceDispatcher_ServiceProxy is deprecated. Please specify an address or serviceNamespaceURI and modify the call accordingly
2 - WebServiceSender attribute [multipartResponse] is deprecated: Unless set explicitly multipart response will be detected automatically

### CODE REVIEW

File: test/src/main/configurations/HTTP/ConfigurationWebServiceListenerSender.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	<adapter name="WebServiceListener" description="Test the functioning of the WebServiceListener">
 		<receiver>
 			<listener className="nl.nn.adapterframework.http.WebServiceListener"
-				name="urn:ws" />
+				name="urn:ws" serviceNamespaceURI="urn:ws" />
```
Comment (nielsm5) [inline test/src/main/configurations/HTTP/ConfigurationWebServiceListenerSender.xml:5]: Could you revert the changes made in this file, as we explicitly test the endpoints here. The other 2 files are ok!

**Review by nielsm5 — APPROVED** [review]

**Review by gvanbrakel — APPROVED** [review]
